components/org.wso2.carbon.identity.api.server.organization.user.sharing.management/org.wso2.carbon.identity.api.server.organization.user.sharing.management.v1/src/main/resources/organization-user-share.yaml

```
@@ -0,0 +1,634 @@
+openapi: 3.0.0
+info:
+  version: 1.0.2
+  title: 'User Share API Definition'
+  description: |-
+    This API provides organization administrators with the ability to manage user access across child organizations. It supports operations to share users with specific or all child organizations, retrieve shared access details, and remove shared access as needed. The API also includes features for paginated retrieval of organizations and roles associated with shared users.
+  contact:
+    name: WSO2
+    url: 'http://wso2.com/products/identity-server/'
+    email: iam-dev@wso2.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  # Added by API Auto Mocking Plugin
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/DinukaBimsaraBodaragama/user-sahred-access-API-4/1.0.2
+  - url: 'https://{server-url}/t/{tenant-domain}/api/server/v1'
+    variables:
+      tenant-domain:
+        default: "carbon.super"
+      server-url:
+        default: "localhost:9443"
+
+paths:
+  /users/share:
+    post:
+      tags:
+        - User Sharing
+      summary: Share a user across specific organizations
+      description: |
+        This API shares one or more users across specified organizations, assigning roles based on the provided policy. The policy defines the sharing scope for each organization, including whether access extends to child organizations.
+        
+        <b>Permission required:</b> `internal_user_shared_access_add`
```

```suggestion
        <b>Scope(Permission) required:</b> `internal_user_shared_access_add`
```

check other places as well

---

Addressed in commit: [cabd548](https://github.com/wso2/identity-api-server/pull/784/commits/cabd548d446c83c21e70c8e955aabd33265c32a0)